### DISCUSSION TITLE
Resend Verification mail

### DISCUSSION THREAD

**Discussion by ValliammaiMuthiah** [Azure/api-management-developer-portal#2598] [Feature request]
**Every bug report should have precise description and reproduction steps; console traces or source code references are appreciated.**

**For assistance requests, contact Azure support or submit a post [on Stack Overflow](http://aka.ms/apimso). We don't provide support through GitHub Issues. Feature requests can be raised [on the Azure Feedback Forum](https://aka.ms/apimwish).**

## Bug description

Once signed up, resend verification feature would be helpful incase of mail is not received properly.

## Reproduction steps

1. Go to Sign up page
2. Fill all the inputs
3. Click the sign up button
4. "Follow the instructions from the email.." message is shown

## Expected behavior

Resend Verification feature will be very helpful incase of mail is not received properly.
Otherwise Account will be created in the backend without verification and for the user, whole sign up process has to be repeated again.

## Is your portal managed or self-hosted?

Self-hosted

## Release tag or commit SHA (if using self-hosted version)

R 18

## Environment

 - Operating system: [e.g., iOS]
 - Browser: [e.g., Google Chrome, Safari]
 - Version: [e.g., 22]

## Additional context

![Resend Verification](https://user-images.githubusercontent.com/75632710/182823236-870fcc8c-2b35-42ed-8a67-b43c3fc5ce0a.PNG)

**Comment by ghost** [comment]
@ValliammaiMuthiah, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@ValliammaiMuthiah, thank you for requesting this feature.

**Comment by Radamonas** [comment]
We would like to have this functionality as well.

**Comment by IanBrown-OAG** [comment]
Would be great to have this in the managed portal code too
